src/pynwb/ndx_patterned_ogen/patterned_ogen.py

```diff
@@ -458,6 +484,25 @@ class PatternedOptogeneticStimulusTable(TimeIntervals):
         },
     )
 
+    @classmethod
+    def check_if_argument_is_list(cls, colset, field_name):
+        for row in range(len(colset[field_name])):
+            if isinstance(colset[field_name][row], (list, np.ndarray, tuple)):
```

Comment:
I have concerns about the use of `instance`; the column definitions about do not defined `dtype` like the `docval` for the helper function does (IDK if that's possible anyway...) but if the logic is the same I think it would be safer to use the logic `not isinstance(<small number of accepted values>)`

Namely, there are a bunch of (technically arbitrarily many) iterable types that could be passed that aren't any of these 3. Most obvious one is an `xarray`, which is not a `np.ndarray`

Whereas for the scalar fields, accepting only `int` or `float` (also, `np.integer` and `np.floating` as generic value types of length 1 array contents)

Reply:
ok changed to only accept `int`, `float`, `np.generic`